documentation/modules/non-admin-permissions-for-ui.adoc

```diff
@@ -0,0 +1,56 @@
+:_content-type: PROCEDURE
+
+[id="non-admin-permissions_{context}"]
+= Granting permissions to non-administrators to work with migration plan components
+
+As an administrator, you can work with all components of migration plans (for example, providers, network mappings, and migration plans).
+
+By default, non-administrators have limited ability to work with migration plans and their components. You can modify their roles to allow them full access to all components or you can give them limited permissions.
```

Comment:
the beginning is good - saying that administrators can do everything and non-admins are limited..

then we need to mention the cluster roles that can be assigned to non-admin, e.g., for migrations: migrations.forklift.konveyor.io-v1beta1-admin, 
migrations.forklift.konveyor.io-v1beta1-edit, and migrations.forklift.konveyor.io-v1beta1-view, and to say that administrator can assign them to users. *-view roles enable to view entities but not to create, delete or modify them. *-edit roles enable to create, delete or modify entities. *-admin roles enable to delete collections of entity resources.

Reply:
later (not for 2.5), we should introduce aggregated roles (like KubeVirt's kubevirt.io:view) and that will simplify this section a bit